package.json

```diff
@@ -12,6 +12,7 @@
     "core-js": "^3.6.5",
     "vue": "^2.6.11",
     "vue-router": "^3.2.0",
+    "vue-slick": "^1.1.20",
```

Comment:
So, on this: 2 points
1. You should have told me before adding this package
2. You didn't need to add this package. Vuetify already has a [carousel component](https://vuetifyjs.com/en/components/carousels/#hide-controls), as well as multiple options for implementing Carousels - e.g. [Slider Groups](https://vuetifyjs.com/en/components/slide-groups/#usage), and [windows](https://vuetifyjs.com/en/components/windows/). The slider group would be the most appropriate for this. If you can't use that then just leave it out.